File: sources/com/lumiyaviewer/lumiya/slproto/chat/_$Lambda$Iyj6QpN_ZLoXueXenKuJvDVzcmI.java
/*
 * Decompiled with CFR 0.152.
 */
package com.lumiyaviewer.lumiya.slproto.chat;

import com.lumiyaviewer.lumiya.slproto.chat.SLChatTextBoxDialog;
import com.lumiyaviewer.lumiya.slproto.users.manager.UserManager;
import com.lumiyaviewer.lumiya.ui.common.TextFieldDialogBuilder;

/*
 * Illegal identifiers - consider using --renameillegalidents true
 */
final class _$Lambda$Iyj6QpN_ZLoXueXenKuJvDVzcmI
implements TextFieldDialogBuilder.OnTextCancelledListener {
    private final Object -$f0;
    private final Object -$f1;

    private final /* synthetic */ void $m$0() {
        ((SLChatTextBoxDialog)this.-$f0).lambda$-com_lumiyaviewer_lumiya_slproto_chat_SLChatTextBoxDialog_4314((UserManager)this.-$f1);
    }

    public /* synthetic */ _$Lambda$Iyj6QpN_ZLoXueXenKuJvDVzcmI(Object object, Object object2) {
        this.-$f0 = object;
        this.-$f1 = object2;
    }

    @Override
    public final void onTextCancelled() {
        this.$m$0();
    }
}

File: lumiya/src/main/java/com/lumiyaviewer/lumiya/slproto/chat/$Lambda$Iyj6QpNZLoXueXenKuJvDVzcmI.kt
package com.lumiyaviewer.lumiya.slproto.chat
import java.util.*

import com.lumiyaviewer.lumiya.slproto.users.manager.UserManager
import com.lumiyaviewer.lumiya.ui.common.TextFieldDialogBuilder

/* renamed from: com.lumiyaviewer.lumiya.slproto.chat.-$Lambda$Iyj6QpN-ZLoXueXenKuJvDVzcmI  reason: invalid class name */
/* synthetic */ class $Lambda$Iyj6QpNZLoXueXenKuJvDVzcmI : TextFieldDialogBuilder.OnTextCancelledListener {

    /* renamed from: -$f0  reason: not valid java name */
    private /* synthetic */ Object f65$f0

    /* renamed from: -$f1  reason: not valid java name */
    private /* synthetic */ Object f66$f1

    private /* synthetic */ Unit $m$0() {
        ((SLChatTextBoxDialog) this.f65$f0).m147lambda$com_lumiyaviewer_lumiya_slproto_chat_SLChatTextBoxDialog_4314((UserManager) this.f66$f1)
    }

    /* synthetic */ $Lambda$Iyj6QpNZLoXueXenKuJvDVzcmI(Object obj, Object obj2) {
        this.f65$f0 = obj
        this.f66$f1 = obj2
    }

    Unit onTextCancelled() {
        $m$0()
    }
}
